packages/core/src/components/essentials/tooltip.ts

```diff
@@ -43,15 +43,21 @@ export class Tooltip extends Component {
 
 				let data = select(event.target).datum() as any;
 
-				// if there is a provided tooltip HTML function
-				if (Tools.getProperty(this.model.getOptions(), "tooltip", "customHTML")) {
-					tooltipTextContainer.html(this.model.getOptions().tooltip.customHTML(data));
-				} else if (e.detail.multidata) {
+				// Generate default tooltip
+				let defaultTooltip;
+				if (e.detail.multidata) {
 					// multi tooltip
 					data = e.detail.multidata;
-					tooltipTextContainer.html(this.getMultilineTooltipHTML(data));
+					defaultTooltip = this.getMultilineTooltipHTML(data);
+				} else {
+					defaultTooltip = this.getTooltipHTML(data);
+				}
+				// if there is a provided tooltip HTML function call it
```

Comment:
suggesting an empty line before the comment (in between the sets of if/else) to maintain consistency with tooltip-bar